01.1-codeofconduct.Rmd

```diff
@@ -0,0 +1,78 @@
+
+#Project-wide Code of Conduct statement for Bioconductor
+[link to code of conduct](https://bioconductor.github.io/bioc_coc_multilingual/)
+
+
+
+(Adapted from the BioC 2020 Code of Conduct)
+Code of Conduct
+Version 1.0.2 (July 27, 2021)
+
+The Bioconductor community values an open approach to science that promotes the
+
+    * sharing of ideas, code, software and expertise
+    * collaboration
+    * diversity and inclusivity
+    * a kind and welcoming environment
+    * community contributions
```

Comment:
Thanks for adding the bullet points here (based on earlier PR and its requests). 

But we have other similar lists later in this text. Well could you add the bullet points in the other relevant places as well?

Reply:
Hi @shigdel - this seems like not resolved yet?